### ISSUE TITLE
NullPointer in PokePointer.OnPostSceneQuery after pressing a button

### ISSUE THREAD

**Issue by julenka** [microsoft/MixedRealityToolkit-Unity#4683]
## Overview
Open the HandInteractionExamples scene and press any button with finger, then move your hand to the right of the button.

After you move your hand off, you will get continuous null pointers, with this call stack:

This is most likely a regression from #4263 

```
NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.Input.PokePointer.OnPostSceneQuery () (at Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs:107)
Microsoft.MixedReality.Toolkit.Input.FocusProvider.UpdatePointer (Microsoft.MixedReality.Toolkit.Input.FocusProvider+PointerData pointer) (at Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs:825)
Microsoft.MixedReality.Toolkit.Input.FocusProvider.UpdatePointers () (at Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs:736)
Microsoft.MixedReality.Toolkit.Input.FocusProvider.Update () (at Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs:475)
```

**Comment by julenka** [comment]
Urgency now because the HandInteractionExamples scene cannot be run.

**Comment by julenka** [comment]
Fixed by #4684
